src/test/scala/uk/gov/hmrc/selenium/webdriver/DriverFactorySpec.scala

```diff
@@ -39,8 +39,9 @@ class DriverFactorySpec extends AnyWordSpec with Matchers with BeforeAndAfterEac
   }
 
   "DriverFactory" should {
-    val accessibilityAssessmentExtension: String =
-      Source.fromResource("browser-extensions/chromium-accessibility-assessment").getLines().mkString
+    val accessibilityAssessmentExtension: String = Base64.getEncoder.encodeToString(
+      getClass.getResourceAsStream("/browser-extensions/chromium-accessibility-assessment.crx").readAllBytes
+    )
```

Comment:
the build is maybe doing a bit less, but it turns out that when you use seleniums `addExtension(File)` - then selenium is just reading it and base64encoding it itself - so we aren't really getting much either way

we've traded a bit less in build.sbt for a bit more in DriverFactory - maybe bit more understandable 🤷‍♂️